src/msix/msix.cpp

```diff
@@ -289,3 +294,66 @@ MSIX_API HRESULT STDMETHODCALLTYPE PackPackage(
 } CATCH_RETURN();
 
 #endif // MSIX_PACK
+
+MSIX_API HRESULT STDMETHODCALLTYPE PackBundle(

```

Comment:
>PackBundle [](start = 35, length = 10)

This must be in the PACK #ifdef. That's why some builds are failing in windows.

Reply:
Updated.